byway/src/main/webapp/home.js

```diff
@@ -0,0 +1,16 @@
+if (document.readyState === 'loading') {  // Loading hasn't finished yet
+  document.addEventListener('DOMContentLoaded', initializeHomePage)
+} 
+else{
+  initializeHomePage();
+}
+
+function initializeHomePage(){
+  fetch('/api/gettrips');
```

Comment:
Replace this pointless fetch statement with a `TODO` comment instead. In general, use TODO comments rather than writing code that doesn't do anything.

Reply:
done